pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,23 @@
+import NextAuth from "next-auth"
```

Comment:
Please add your new env var to env.example

Reply:
I added only NEXTAUTH_SECRET to .env.example.